apps/Github Desktop/description

```diff
@@ -0,0 +1,5 @@
+Github Desktop is a git GUI client.
```

Comment:
```suggestion
Github Desktop is a GUI git client.
```

Reply:
It's more a github gui client than a git one.